pyproject.toml

```
@@ -29,9 +29,10 @@ flake8-string-format = "~=0.3"
 flake8-tidy-imports = "~=4.1"
 flake8-todo = "~=0.7"
 pep8-naming = "~=0.11"
+pip-licenses = "~=3.5.2"
 pre-commit = "~=2.1"
-taskipy = "^1.6.0"
 python-dotenv = "^0.15.0"
+taskipy = "^1.6.0"
```

```suggestion
python-dotenv = "~=0.15"
taskipy = "~=1.6"
```
Shouldn't these be pinned like other deps for consistency, and I realise this is not related to your PR :P

---

Perhaps, but that's is indeed out of scope :P